client/src/components/userAuctions/UserAuctions.jsx

```diff
@@ -36,7 +36,72 @@ class UserAuctions extends Component {
   }
 
   render() {
-    const { openAuctions, closedAuctions, isFetching } = this.props.userAuctions;
+    let { openAuctions, closedAuctions, isFetching } = this.props.userAuctions;
+    closedAuctions = [
```

Comment:
these are for development only. Once we have the worker populating the closed_auctions table we won't need this. You'll see that we do actually have data from the server coming in, but it's not parsing out the closed auctions because the table is empty.